sessionctx/variable/sysvar.go

```diff
@@ -372,7 +374,16 @@ var defaultSysVars = []*SysVar{
 		GlobalLogMaxDays.Store(int32(maxAge))
 		cfg := config.GetGlobalConfig().Log.ToLogConfig()
 		cfg.Config.File.MaxDays = int(maxAge)
-		err = logutil.ReplaceLogger(cfg)
+
+		opt := zap.WrapCore(func(core zapcore.Core) zapcore.Core {
+			keyspaceName := keyspace.GetKeyspaceNameBySettings()
+			if !keyspace.IsKeyspaceNameEmpty(keyspaceName) {
+				core = core.With([]zap.Field{zap.String("keyspaceName", keyspaceName)})
+			}
+			return core
+		})
```

Comment:
Let's extract this code snippet to a function `WithKeyspace` in `util/logutil`:
```go
func WithKeyspace(core zapcore.Core) zapcore.Core {
	keyspaceName := keyspace.GetKeyspaceNameBySettings()
	if !keyspace.IsKeyspaceNameEmpty(keyspaceName) {
		core = core.With([]zap.Field{zap.String("keyspaceName", keyspaceName)})
	}
	return core
}
```

So we can reuse this function here and in `tidb-server/main.go`: 
```go
opt := zap.WrapCore(logutil.WithKeyspace)
```

Reply:
yesh, thanks your suggestion. PTAL~